### PR TITLE
Change to new Kano World API URLs

### PR DESCRIPTION
**Pull request by skarbat**
Set new default URLs for Kano World in production mode.
Related PR: https://github.com/KanoComputing/kano-apps/pull/138

### CODE REVIEW

File: kano_world/config.py [hunk 1]
```diff
@@ -22,10 +22,10 @@ def load_conf():
         conf = {}
 
     if 'api_url' not in conf:
-        conf['api_url'] = 'https://api.kano.me'
+        conf['api_url'] = 'https://apps-directory.os.kano.me'
```
Comment (radujipa) [inline kano_world/config.py]: `https://worldapi.kano.me`

File: kano_world/config.py [hunk 2]
```diff
@@ -22,10 +22,10 @@ def load_conf():
...
 
     if 'world_url' not in conf:
-        conf['world_url'] = 'http://world.kano.me'
+        conf['world_url'] = 'http://apps-directory.os.kano.me'
```
Comment (radujipa) [inline kano_world/config.py]: This shouldn't change.

**Comment by skarbat** [comment]
Superseede by: https://github.com/KanoComputing/kano-profile/pull/470/files

**Comment by skarbat** [comment]
Removing this one, it's obsoleted by Radu's PR - thanks!

**Pull request closed without merging**